pkg/application/consul/options.go

```diff
@@ -0,0 +1,49 @@
+package consul
+
+import (
+	"github.com/awslabs/eksdemo/pkg/application"
+	"github.com/awslabs/eksdemo/pkg/cmd"
+)
+
+type ConsulOptions struct {
```

Comment:
If you just rename this to `Options` it should pass the linter.

Reply:
Went with AppOptions, passed local golangci-lint run.